### DISCUSSION TITLE
X86 OLED screen with GPIO buttons

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#1241] [Ideas]
It'd be cool to get this working with myMPD. Source for idea below.  Thought of having a nice, audio receiver like case, with just this. Then breakout the USB power and data to two different headers to be able to supply separate power. Would basically be a dumb terminal to the Thin Client I run myMPD on. Using the GPIO device they mention, to add buttons. Just not sure what to do Linux kernel headers wise. Obviously not using Volumio, just Debian 12.

Volumio forums:
https://community.volumio.com/t/dev-plugin-mpd-oled-for-x86-based-systems/59020

Github:
https://github.com/wheaten/mpd_oled_dev/blob/main/doc/X86-install_volumio3_source.md

**Comment by jcorporation** [comment]
> Using the GPIO device they mention, to add buttons.

You can use https://github.com/jcorporation/myGPIOd to add buttons to control MPD and myMPD.

**Comment by bonelifer** [comment]
Should make it clear, we're talking about a usb based gpio board for x86.

**Reply by jcorporation** [reply]
If the Linux kernel and libgpiod support it, it should work with myGPIOd.
